bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryserviceplugin.js

```diff
@@ -63,7 +63,9 @@ YUI.add('cof-createcontent-universaldiscoveryserviceplugin', function (Y) {
                     title: this.get('discoveryWidgetTitle'),
                     multiple: false,
                     contentDiscoveredHandler: Y.bind(this._setSelectedLocation, this, target),
-                    isSelectable: function () {return true;},
+                    isSelectable: function (contentStruct) {
```

Comment:
I prefer to have this bundle ES6 free. As far as I know the last version of this bundle is installed in every eZPlatfrom version and in <1.7 we supported some browsers which don't support ES6 correctly.